### DISCUSSION TITLE
Using the gum filter, present a list that shows both the preselected and selected items

### DISCUSSION THREAD

**Discussion by bdrnglm** [charmbracelet/gum#341] [Q&A]
Currently, I'm displaying a list of items using the gum filter along with the --no-limit flag.
However, I need to have specific items preselected, and I haven't been able to find an option for that.
Is there a way to do it or should I create an issue for that?

**Comment by MikaelFangel** [comment]
It is not implemented, but could what your suggesting be covered by #337 ?

**Reply by bdrnglm** [reply]
Not really, my concern is not about the cursor position, but rather the ability to preselect specific items using the gum filter.

**Reply by MikaelFangel** [reply]
Maybe I'm misunderstanding something but isn't also what's meant here with adding selected?

"Adding an additional flag also allows combining these flags together for even finer control.
Using --selected and --indicator-value together allows listing all the "default" options first pre-selected, and then starting the indicator after all the default options, so one doesn't have to move through all the default options first." - quote #337

**Reply by bdrnglm** [reply]
The information previously mentioned pertains to the `--selected` option of the `gum choose` command.
However, I am referring to a similar option that applies to the `gum filter` command.

Thank you for bringing up issue #337, as I was unaware of the existence of the `--selected` flag.
Would it be beneficial for me to revise my original message (and its title) to enhance clarity?
